### PR TITLE
Update README with instructions for vocabularies & fix issue #5

### PR DESCRIPTION
**Pull request by keesvanbochove**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -243,7 +243,6 @@ networks:
     driver: bridge
     internal: false
   api:
-    name: api
```
Comment (blootsvoets) [inline docker-compose.yml:246]: As mentioned in #5, please keep this as it was.

**Comment by blootsvoets** [comment]
Incompatibility between docker-compose files was fixed in 95ea7c4.

**Comment by blootsvoets** [comment]
Merged with #7.

**Comment by keesvanbochove** [comment]
Thanks for merging the comments @blootsvoets, and #5 is fixed now indeed.

**Pull request closed without merging**